domain/src/memory.js

```diff
@@ -86,6 +88,17 @@ function selectNotDeleted(dict) {
 }
 
 /**
+ * Полностью обновляет предмет(кроме id)
+ * @returns {Promise.<Number, Error>} Количество найденных объектов
+ */
+async function updateItemFully(db, coll, id, item) {
+    if (await database.get_by_id(db, coll, id)) {
+        await database.add_by_id(db, coll, id, item);
+        return 1;
+    } else
+        return 0;
```

Comment:
The return value is logically a boolean value, so just return boolean (true/false).

Moreover, if you return integer like that, it usually means that you return error code, and usual return code for success is 0. Because it is often the case that there is single "happy" path and a lot of "error" paths - so "happy" path is special in that sense, and 0 is also special, so 0 for "happy" path makes sense - e.g. for all console programs return value of 0 means that program successfully finished execution.